src/error.rs

```diff
@@ -75,6 +80,13 @@ impl fmt::Display for FormatterError {
             Self::Internal(message, _) | Self::Query(message, _) => {
                 write!(f, "{message}")
             }
+            Self::Formatting(err) => {
```

Comment:
It is associated with idempotence check, but if the output produced by topiary is not parsable, it has nothing to do with idempotence. I used `Formatting` because it was the one you suggested in https://github.com/tweag/topiary/issues/106#issuecomment-1341003683 and it sounded quite accurate to me. No strong opinion on which word should be used.

Reply:
I don't have a very strong opinion either. I agree that the error is a formatting error, but the resulting error message is associated with the idempotency check. So if someone decided to throw a `Formatting(err)` in the future, they would get an error message about idempotency.